packages/advanced-logic/test/utils/payment-network/erc777/stream-create-data-generator.ts

```diff
@@ -197,6 +234,48 @@ export const requestFullStateCreated: RequestLogicTypes.IRequest = {
   version: '0.1.0',
 };
 
+export const subsequentRequestFullStateCreated: RequestLogicTypes.IRequest = {
+  creator: {
+    type: IdentityTypes.TYPE.ETHEREUM_ADDRESS,
+    value: TestData.payeeRaw.address,
```

Comment:
Can you please take the opportunity to refactor a bit? Test cases should not be overly complex but for dumb data, it's worth it a factorization. This identity is used 8 times on this file, for instance. It's not only handy for subsequent developers to read, it also helps focusing on what changes from test case to test case.